### ISSUE TITLE
Feature request Right Sidebar > Emojis-Stickers-Gifs/UserInfo-GroupInfo

### ISSUE THREAD

**Issue by Acid-Crash** [UnigramDev/Unigram#554]
Regular Telegram desktop App has so called Right Sidebar.
It acts as the panel for Emojis-Stickers-Gifs and as UserInfo-GroupInfo.
![telegramrightsidebar](https://user-images.githubusercontent.com/32600318/37062576-878f4260-219f-11e8-9f89-63693ffe01f4.png)

In Emojis-Stickers-Gifs mode it switches between Emojis, Stickers, and Gif Searcher (with recently used gifs)
In UserInfo-GroupInfo mode it grants access to the shared media, members list etc within selected chat/group.
The feature allows to get quick access to common without losing sight of the actual conversation (it remains visible) which may be useful for a messaging app, especially on desktop wide screens configs.
Taking into account that Unigram relies on responsive design this feature may be implemented as toggler (show/hide switcher) or become active when app width becomes higher then predefined number.

**Comment by joshuader6** [comment]
Especially the Emoji-bar would be useful.
Maybe I haven't found it but there is no "simple" way to insert emojis.

**Comment by joshuader6** [comment]
@DarthJahus Sadly this does only work with the en_US locale. See [here](https://mspoweruser.com/heres-new-emoji-picker-windows-10-might-not-work/)

**Comment by DarthJahus** [comment]
Spring Creators Update will bring it to most (if not all) locales. It's planned to release on April.
That said, the slow ring build (17120) is pretty stable and enables that feature.

**Comment by MarcAnt01** [comment]
Now, it's available on every language, you can add emoji with emoji panel with Windows + on PC and on mobile or Windows Tablet Mode through keyboard.
To get pieces of information about group and members, click on group name area will give you all the required stuff

**Comment by FrayxRulez** [comment]
Implemented emojis panel in https://github.com/UnigramDev/Unigram/commit/a0560413a837fbf7b800dbe6e034b8e634e349d0, not as a sidebar tho

**Comment by rromm** [comment]
NEED SIDEBAR
very uncomfortable to open stickers/gifs frequently

**Comment by Xplorer4x4** [comment]
Please bring back the right sidebar!

**Comment by MarcAnt01** [comment]
> Please bring back the right sidebar!

Never been in Unigram, actually

**Comment by Xplorer4x4** [comment]
> > Please bring back the right sidebar!
> 
> Never been in Unigram, actually

Typo but the idea is obviously the same!

**Comment by MarcAnt01** [comment]
Opening it again, but not sure if we can see this feature in Unigram

**Comment by MarcAnt01** [comment]
![image](https://user-images.githubusercontent.com/43090263/81668244-ada24000-9444-11ea-952b-bc19bcf733a3.png)

**Comment by MarcAnt01** [comment]
Emoji/sticker/gifs has been added in the source code (might take a bit to reach the store version)

**Comment by rromm** [comment]
> (might take a bit to reach the store version)

We are  still waiting, let us know in this topic when it's available.

**Comment by FrayxRulez** [comment]
> We are still waiting, let us know in this topic when it's available.

You can grab a beta from t.me/unigramappx

**Comment by dada051** [comment]
We've got the panel at the right side with Unigram 4.x, but just for pinning chat information only useless (for me) icons, that I can add with keyboard shortcuts :(

**Comment by TinyAtoms** [comment]
Yeah, I prefer the UserInfo-GroupInfo compared to the emoji/sticker thing. I honestly didn't know that telegram had the sticker sidebar, I never use it.

**Comment by sethazazel** [comment]
Info sidebar is really usefull. I am always using it while i am searching for files, links or pictures. It would be good if you add it. It is only reason that i am still using telegram desktop. (and emoji sidebar is a little bit annoying for me)

**Comment by MarcAnt01** [comment]
@sethazazel you can disable emoji sidebar

**Comment by yada-yada-git** [comment]
@MarcAnt01 How do you disable the emoji sidebar and show Group info?

**Comment by MarcAnt01** [comment]
@yada-yada-git you cannot

**Comment by foal** [comment]
Hi, how to disable/hide the emoji sidebar?

**Comment by FrayxRulez** [comment]
You can click on the emoji button next to the send button

**Comment by dada051** [comment]
Click on the emoji icon in the message input.

**Comment by foal** [comment]
O_o!!! Thank you! It will be nice to hide it automatically on read-only channels. Because they haven't this button at all. Switch to RW channel, click on the emoji button and switch back works, but it not so clear :)

**Comment by deshario** [comment]
@dada051 Thanks

**Comment by CCCAUCHY** [comment]
Does anyone else remember this issue?

**Comment by clansty** [comment]
![image](https://github.com/user-attachments/assets/46884c6d-d7fc-4445-8d4c-20cf7925d96c)

**Comment by Mordsdima** [comment]
im sorry but im using ms store version and i cant open sidebar :/
even tho im using latest version (11.13 X64)
